src/backend/distributed/master/master_delete_protocol.c

```diff
@@ -268,8 +268,13 @@ master_drop_sequences(PG_FUNCTION_ARGS)
 		appendStringInfo(dropSeqCommand, " %s", TextDatumGetCString(sequenceText));
 	}
 
-	SendCommandToWorkers(ALL_WORKERS, DISABLE_DDL_PROPAGATION);
-	SendCommandToWorkers(ALL_WORKERS, dropSeqCommand->data);
+	if (dropSeqCommand->len != 0)
+	{
+		appendStringInfoString(dropSeqCommand, " CASCADE");
```

Comment:
There are multiple solutions to avoid this problem:
- Try to clean a worker on `stop_metadata_sync_to_node`
- Propagate `DROP TABLE` to all workers, instead of just metadata workers
- `DROP SEQUENCE` with `CASCADE` option